sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/ReceiptItem.java

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.formrecognizer.models;
+
+import com.azure.core.annotation.Fluent;
+
+/**
+ * The ReceiptItem model.
+ */
+@Fluent
+public class ReceiptItem {
+
+    /*
+     * Name of the field value.
+     */
+    private FieldValue<?> name;
```

Comment:
Are these left generic so that the service could return an integer value here? It seems that these APIs will need a lot of type checking to be done to be used safely or to have a lot of unchecked casts.

Reply:
Right, so `FieldValue` here is an abstract class that can take in a `T` that could be of type Integer, String, Float, Date, Time atm.
It is to accommodate the `valueDate`, `valueTime`, `valuePhoneNumber` properties on Service model FieldValue.